app/res/values-de/strings.xml

```diff
@@ -252,8 +252,8 @@
     <string name="code">Code</string>
     <string name="following_user">Folgen…</string>
     <string name="unfollowing_user">Entfolgen…</string>
-    <string name="starring_repository">Hervorheben…</string>
-    <string name="unstarring_repository">Hervorhebung entfernen…</string>
+    <string name="starring_repository">Favorisiere Repository…</string>
+    <string name="unstarring_repository">Defavorisiere Repository…</string>
```

Comment:
If we do like so. We must use Forking Repository and Lösche Repository too... 
(same with gists :-))